pcsc-sys/build.rs

```diff
@@ -20,15 +20,24 @@ fn main() {
         }
 
         _ => {
-            pkg_config::Config::new()
-                .atleast_version("1")
-                .probe("libpcsclite")
-                .expect(&format!(
-                    r#"Could not find a PCSC library.
+            let lib_dir = env::var("PCSCLITE_LIB_DIR");
+            let include_dir = env::var("PCSCLITE_INCLUDE_DIR");
```

Comment:
We don't use any header files, we just link. Does things work if you remove this?

Reply:
Good point. I think it should work. I test it and modify if it works.

Reply:
It works :)